flow360/component/simulation/web/asset_base.py

```diff
@@ -31,21 +31,23 @@ class AssetBase(metaclass=ABCMeta):
     _entity_info: EntityInfoModel = None
 
     # pylint: disable=redefined-builtin
-    def __init__(self, id: str):
+    def __init__(self, id: str = None):
```

Comment:
Let's not add default for id.

It does not bring any convenience but makes the interface more error-prone.

Also the default/most likely use case should not be when id is None.

```suggestion
    def __init__(self, id: Union[str, None]):
```

Reply:
Add in docstring that when id is none the asset is meant to be operating in local mode.

Reply:
Updated and also added the docstring